components/pipeline-service/base/rbac/cluster-role/pipeline-maintainer.yaml

```diff
@@ -0,0 +1,60 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pipeline-maintainer
+rules:
+  - apiGroups:
+      - operators.coreos.com
+    resources:
+      - installplans
+    verbs:
+      - get
+      - list
+      - update
+      - patch
+  - verbs:
+      - patch
+      - get
+    apiGroups:
+      - ''
+    resources:
+      - serviceaccounts
+    resourceNames:
+      - pipeline # TODO: figure out how to 'gitops' this.
+  - verbs:
+      - create
+      - get
+      - list
+      - watch
+      - delete
+    apiGroups:
+      - ''
+    resources:
+      - secrets
+  - verbs:
+      - '*' # needed till we figure out how to cleanup workspaces.
+    apiGroups:
+      - 'tekton.dev'
+    resources:
+      - 'pipelineruns'
+  - apiGroups:
+      - results.tekton.dev
+    resources:
+      - results
+      - records
```

Comment:
I know it was not present in https://github.com/redhat-appstudio/infra-deployments/blob/main/components/authentication/base/component-maintainer.yaml but `logs` should be added to the resource list @manish-jangra 

Can you do that as part of creating this file?

thanks

@sayan-biswas FYI

Reply:
It should now be added into `pipeline-maintainer.yaml`, though I have added in `component-maintainer.yaml`
```YAML
- apiGroups:
    - results.tekton.dev
  resources:
    - results
    - records
    - logs
  verbs:
    - get
    - list
```